engines/scumm/saveload.cpp

```diff
@@ -68,7 +68,7 @@ struct SaveInfoSection {
 
 #define SaveInfoSectionSize (4+4+4 + 4+4 + 4+2)
 
-#define CURRENT_VER 102
+#define CURRENT_VER 106
```

Comment:
why 106? shouldn't be 103?

Reply:
During the months of development there have been four internal savegames revisions.
Should I change that to 103?

Reply:
for final PR yes, but you can keep it for now until PR is ready to merge

Reply:
Very good, I'll let this conversation open for now then, let me know when it's time to change this then